src/Mantasflowers.Services/FirebaseService/GetTokensResponse.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Newtonsoft.Json;
+
+namespace Mantasflowers.Services.FirebaseService
```

Comment:
Normally, this is what `Contracts` folder is for. Maybe an exception could be made if this was used internally between us and firebase, but this is actually exposed to the client of our API.